src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -1883,6 +1886,38 @@ public void canPageAllItems() throws InterruptedException, ExecutionException, T
     assertThat(secondPage.getInteger("totalRecords"), is(5));
   }
 
+  @Test
+  public void canFetchItemsViaPost() throws InterruptedException, ExecutionException, TimeoutException {
+    UUID holdingsRecordId = createInstanceAndHolding(MAIN_LIBRARY_LOCATION_ID);
+
+    List<String> itemIdz = new ArrayList<>();
+    int numOfItemsToCreate = 200;
```

Comment:
Please reduce the number of items to create, as big numbers will increase test execution.
The API is not about a large number of items to retrieve but about the usage of CQL via POST. So, it will be enough to check on 5 items maximum.
Also the test doesn't check how offset and limit fields work, probably a separate test needed.